code/game/turfs/open.dm

```diff
@@ -910,10 +910,104 @@
 	allow_construction = FALSE
 	supports_surgery = FALSE
 
+/turf/open/shuttle/can_surgury
```

Comment:
you have misspelled the word "surgery"

Reply:
Will renaming it require me to update all the tiles manually in the maps?

Reply:
strongdmm has an inbuilt tool to convert non-existent typepaths if you open up a map that has one